public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx

```diff
@@ -40,6 +41,11 @@ export const TimeSeriesPanel: React.FC<TimeSeriesPanelProps> = ({
   const frames = useMemo(() => prepareGraphableFields(data.series, config.theme2, timeRange), [data, timeRange]);
   const timezones = useMemo(() => getTimezones(options.timezone, timeZone), [options.timezone, timeZone]);
 
+  let matchByLabel = 'pod';
```

Comment:
this should be replaced with whatever logic we have in place that matches an exemplar field with each field's `field.name`, or some other more complex match-up logic, like field name + field.label(s).